README.md

```diff
@@ -80,6 +80,7 @@
 - [stylelint-formatter-github](https://www.npmjs.com/package/stylelint-formatter-github) - Output errors and warnings directly in GitHub pull requests.
 - [stylelint-formatter-gitlab-code-quality-report](https://www.npmjs.com/package/stylelint-formatter-gitlab-code-quality-report) - Output errors and warnings as a GitLab Code Quality report (artifact).
 - [stylelint-formatter-pretty](https://www.npmjs.com/package/stylelint-formatter-pretty) - A pretty formatter for terminals.
+- [@csstools/stylelint-formatter-github](https://www.npmjs.com/package/@csstools/stylelint-formatter-github) - Output errors and warnings directly in GitHub pull requests.
```

Comment:
[suggestion] Can you move this to the top of the list in Formatters? Packages starting with `@` are ordered first.